LSP/Example4.md

```diff
@@ -0,0 +1,23 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/microsoft/vscode/blob/25be28623b51c5e3acf1a0d9ac1854c196ec4d37/src/vs/editor/browser/viewParts/marginDecorations/marginDecorations.ts
```

Comment:
➖
There are different render methods. I'm not sure about abstract class and LSP, because we can implement some methods in an abstract class and use them, so basically we can rewrite them. But here render and _render are look good.

Reply:
👎 instance of abstract class can't be created